From f206d925c5556630804216e49d38ba99da176877 Mon Sep 17 00:00:00 2001
From: Dmitry Artamonow <mad_soft@inbox.ru>
Date: Wed, 4 Mar 2009 20:22:52 +0300
Subject: [PATCH 17/28] h3600: remove unused set_wake callback from sa1100_port_fns

It's unused by serial core since 2.5.x era and h3600 seem to be the only
machine that sets it.

See commit f934c8688171ec537f7a83d79e55ec930d1c25ce in old-2.6-bkcvs.git

Signed-off-by: Dmitry Artamonow <mad_soft@inbox.ru>
---
 arch/arm/mach-sa1100/h3600.c |   19 -------------------
 1 files changed, 0 insertions(+), 19 deletions(-)

diff --git a/arch/arm/mach-sa1100/h3600.c b/arch/arm/mach-sa1100/h3600.c
index a2670be..96da4e1 100644
--- a/arch/arm/mach-sa1100/h3600.c
+++ b/arch/arm/mach-sa1100/h3600.c
@@ -104,29 +104,10 @@ static void h3600_uart_pm(struct uart_port *port, u_int state, u_int oldstate)
 		assign_h3600_egpio(IPAQ_EGPIO_RS232_ON, !state);
 }
 
-/*
- * Enable/Disable wake up events for this serial port.
- * Obviously, we only support this on the normal COM port.
- */
-static int h3600_uart_set_wake(struct uart_port *port, u_int enable)
-{
-	int err = -EINVAL;
-
-	if (port->mapbase == _Ser3UTCR0) {
-		if (enable)
-			PWER |= PWER_GPIO23 | PWER_GPIO25; /* DCD and CTS */
-		else
-			PWER &= ~(PWER_GPIO23 | PWER_GPIO25); /* DCD and CTS */
-		err = 0;
-	}
-	return err;
-}
-
 static struct sa1100_port_fns h3600_port_fns __initdata = {
 	.set_mctrl	= h3600_uart_set_mctrl,
 	.get_mctrl	= h3600_uart_get_mctrl,
 	.pm		= h3600_uart_pm,
-	.set_wake	= h3600_uart_set_wake,
 };
 
 static struct map_desc h3600_io_desc[] __initdata = {
-- 
1.6.1.3

